flexport/latitude
# Banner Close Button icon no longer aligns right

## Description
This PR resolves the issues with the Banner Close Icon Button when the X icon would align right within its button.

## Screenshots
Before: 
![image](https://user-images.githubusercontent.com/5897853/71857068-fadb6300-309a-11ea-9aca-c6ef53c4939c.png)

After:
![image](https://user-images.githubusercontent.com/5897853/71856989-aafc9c00-309a-11ea-911d-e1a6db6bf160.png)


## Manual Testing Plan
Open storybook and go to the Banner stories http://localhost:6006/?path=/story/latitude-interaction-banner--banner-intents
Observe that the close icon no longer aligns right within the button
Go to the buttons story
See that button examples are unchanged
